django/contrib/postgres/fields/array.py

```diff
@@ -179,6 +180,14 @@ def as_sql(self, qn, connection):
 
 
 @ArrayField.register_lookup
+class ArrayExact(Exact):
+    def as_sql(self, qn, connection):
+        sql, params = super(ArrayExact, self).as_sql(qn, connection)
+        sql += '::%s' % self.lhs.output_field.db_type(connection)
```

Comment:
NVM then, I didn't notice it was a pattern used around the file. Let's be consistent here.